site2/docs/admin-api-clusters.md

```diff
@@ -83,8 +83,8 @@ Here's an example cluster metadata initialization command:
 ```shell
 bin/pulsar initialize-cluster-metadata \
   --cluster us-west \
-  --zookeeper zk1.us-west.example.com:2181 \
-  --configuration-store zk1.us-west.example.com:2184 \
+  --metadata-store zk1.us-west.example.com:2181 \
```

Comment:
we'd better use `--metadata-store zk:zk1.us-west.example.com:2181` to clarify use zookeeper as meta store. 
The same below

Reply:
Thanks for your suggestion. Done.